docs/installation/installation-using-docker-linux.md

```diff
@@ -16,6 +16,8 @@ cd project-base
 *Note: The `--no-install` option disables installation of the vendors - this will be done later in the Docker container.*
 
 *Note: The `--keep-vcs` option initializes GIT repository in your project folder that is needed for diff commands of the application build and keeps the GIT history of `shopsys/project-base`.*
+
+*Note: The `--stability=alpha` option enables you to install the project from the last alpha release. Default value for the option is `stable` but there is no stable release yet.*
```

Comment:
Consider reformulating notes into one:

*Notes:*
* *The `--no-install` option disables installation of the vendors - this will be done later in the Docker container.*
* *The `--keep-vcs` option initializes GIT repository in your project folder that is needed for diff commands of the application build and keeps the GIT history of `shopsys/project-base`.*
* *The `--stability=alpha` option enables you to install the project from the last alpha release. Default value for the option is `stable` but there is no stable release yet.*

Also in other files.

If you will think the current state is fine, then don't do any changes.

Reply:
What exactly is the change? Diff would be nice.

Reply:
I have a feeling that three notes are easier to read when they are a list. Also the triple *Note:* is reduced to one

Reply:
Ah, so only design point.